base/REPL.jl

```diff
@@ -771,9 +771,10 @@ function setup_interface(repl::LineEditREPL; hascolor = repl.hascolor, extra_rep
                                               :help  => help_mode))
     if repl.history_file
         try
-            f = open(find_hist_file(), true, true, true, false, false)
+            hist_path = find_hist_file()
+            f = open(hist_path, true, true, true, false, false)
```

Comment:
Not sure what you mean. It gets closed when the `replc` gets garbage collected. When do you want it to be closed?